pipelines/batch/src/main/java/org/openmrs/analytics/JdbcFetchHapi.java

```diff
@@ -148,11 +149,12 @@ public FetchRowsJdbcIo(
       this.dataSourceConfig = dataSourceConfig;
       // Note the constraint on `res.res_ver` ensures we only pick the latest version.
       StringBuilder builder =
-          new StringBuilder(
-              "SELECT res.res_id, res.res_type, res.res_updated, res.res_version, res.res_ver, "
-                  + "ver.res_encoding, ver.res_text FROM hfj_resource res, hfj_res_ver ver "
-                  + "WHERE res.res_type = ? AND res.res_id = ver.res_id AND "
-                  + "res.res_ver = ver.res_ver AND res.res_id % ? = ? ");
+              new StringBuilder(
```

Comment:
nit: I think the indents are not right here; can you please run `mvn clean package` and update the PR; that should take care of formatting issues like this. BTW, some of the unit-tests seem to be failing; please fix those as well.

Reply:
Done